src/main/java/io/druid/data/input/impl/JSONParseSpec.java

```diff
@@ -19,26 +19,42 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParser.Feature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.metamx.common.parsers.JSONParser;
 import com.metamx.common.parsers.Parser;
 
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 /**
  */
 public class JSONParseSpec extends ParseSpec
 {
   private final ObjectMapper objectMapper;
+  private final Map<String, Boolean> featureSpecs;
 
   @JsonCreator
   public JSONParseSpec(
       @JsonProperty("timestampSpec") TimestampSpec timestampSpec,
-      @JsonProperty("dimensionsSpec") DimensionsSpec dimensionsSpec
+      @JsonProperty("dimensionsSpec") DimensionsSpec dimensionsSpec,
+      @JsonProperty("featureSpec") Map<String, Boolean> featureSpecs
```

Comment:
Can this be changed to be consistently `featureSpec` and drop that extra s on the variable name?